CHANGELOG.md

```diff
@@ -7,8 +7,7 @@
 * None
 
 ### Fixed
-* <How to hit and notice issue? what was the impact?> ([#????](https://github.com/realm/realm-js/issues/????), since v?.?.?)
-* None
+* Callbacks during client reset were not called. ([#6201](https://github.com/realm/realm-js/issues/6201), since v12.0.0)
```

Comment:
I think it can make it a bit clearer if the exact callbacks are listed 🙂 
```suggestion
* The `onBefore`, `onAfter`, and `onFallback` client reset callbacks were not called. ([#6201](https://github.com/realm/realm-js/issues/6201), since v12.0.0)
```